print_designer/print_designer/page/print_designer/jinja/main.html

```diff
@@ -164,6 +175,11 @@
             <div style="width: 100%; position: relative; top:0px; left: 0px; height:{{ settings.page.footerHeightWithMargin }}px; ">{% if footerElement %}{{ renderFooter }}{%endif%}</div>
             </div>
     </div>
+<script>
+	/*! JsBarcode v3.11.5 | (c) Johan Lindell | MIT license */
```

Comment:
Not sure if this is the best way to load the library into wkhtmltopdf 😅

Reply:
have u tried with WeasyPrint?
https://github.com/frappe/erpnext/issues/35155